pdr_backend/lake/etl.py

```diff
@@ -224,12 +286,14 @@ def _calc_bronze_start_end_ts(self) -> Tuple[UnixTimeMs, UnixTimeMs]:
             ETL updates should use to_timestamp by calculating
             min(max(source_tables_max_timestamp)).
         """
-        from_timestamp = self.get_timestamp_values(
-            self.bronze_table_names, self.ppss.lake_ss.st_timestr
+        st_timestr = self._get_min_timestamp_values_from(
+            [NamedTable(tb) for tb in self.gql_data_factory.raw_table_names]
```

Comment:
if you get min_timestamp from raw_tables... you're going to get the very first event... which means you'll be re-processing all events historically. 

This is NOT what you want to do.

Reply:
First run:
Using the `ppss.lake_ss.st_ts` is the exact problem because it can either be fix or dynamic
- if the values is a fixed date than not problem, all tables are getting the same start date
- if value is dynamic, like 18d ago than the value is different whenever you use it and you will end up with different start dates.
Ex: gql start at 2024.05.19_11:00 -> st_date 2024.05.1_11:00
      etl start at  2024.05.19_11:10 -> st_date 2024.05.1_11:10
     -> 10min gap between the start dates
     
Second/third:
You got it wrong, the start date is read from ppss just for the first run when there are no rows in the bronze tables, in all the other cases it gets the start time from the bronze tables.
This was not changed, the only thing changes is that on the first run when there is no data in the bronze tables it gets the start date from raw tables instead of ppss, and the reason is explain above ( the ppss can be dynamic )